com.hazelcast.internal.jmx.LockMBeanTest.testLockCountIsTwo_whenLockedTwice

https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/14217

```
java.lang.AssertionError: expected:<2> but was:<1>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.internal.jmx.LockMBeanTest.testLockCountIsTwo_whenLockedTwice(LockMBeanTest.java:71)
```

--- a/hazelcast/src/test/java/com/hazelcast/internal/jmx/LockMBeanTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/internal/jmx/LockMBeanTest.java
@@ -62,8 +62,8 @@ public class LockMBeanTest extends HazelcastTestSupport {
 
     @Test
     public void testLockCountIsTwo_whenLockedTwice() throws Exception {
-        lock.lock(1000, TimeUnit.MILLISECONDS);
-        lock.lock(1000, TimeUnit.MILLISECONDS);
+        lock.lock(10000, TimeUnit.MILLISECONDS);
+        lock.lock(10000, TimeUnit.MILLISECONDS);
 
         // check number of times locked (locked twice now)
         int lockCount = getIntegerAttribute("lockCount");

